grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -28,7 +28,7 @@ def section_actor_info(panels):
                           [panels.table_target(f"{metric('actor_info')}")], excluded_cols),
         panels.table_info("Materialized View Info",
                           "Mapping from materialized view table id to it's internal table ids",
-                           [panels.table_target(f"{metric('materialized_info')}")], excluded_cols),
+                           [panels.table_target(f"group({metric('table_info')}) by (materialized_view_id, table_id, table_name, table_type)")], excluded_cols),
```

Comment:
Not related to this PR: how about showing finer grain information by including the actor_id for this panel in the dev dashboard? In other words, don't do the group by here in dev dashboard. This can be useful for debugging when we want to associate streaming actor metric with hummock table metric.

Reply:
Thus this panel will contain too much information, especially when there are too many mvs. But this is for dev, so I can accept both.

Reply:
Or just add another panel `table_info` back.